<ui version="4.0" >
 <class>Backend::Ui::ConfigWidget</class>
 <widget class="QWidget" name="Backend::Ui::ConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QGroupBox" name="video_group" >
     <property name="title" >
      <string>비디오</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>출력 드라이버</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="vo_combo" />
        </item>
        <item>
         <spacer name="horizontalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="audio_group" >
     <property name="title" >
      <string>오디오</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>출력 드라이버</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ao_combo" />
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" native="1" name="volume_amp_widget" >
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>볼륨 증폭률</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="volume_amp_spin" >
           <property name="accelerated" >
            <bool>true</bool>
           </property>
           <property name="suffix" >
            <string> %</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
